@here/harp-omv-datasource/lib/OmvPointOfViewFeatureModifier.ts

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (C) 2017-2019 HERE Europe B.V.
+ * Licensed under Apache 2.0, see full license in LICENSE
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { MapEnv } from "@here/harp-datasource-protocol/index-decoder";
+import { LoggerManager } from "@here/harp-utils";
+import { OmvGenericFeatureModifier } from "./OmvDataFilter";
+import { OmvFeatureFilterDescription, OmvFilterDescription } from "./OmvDecoderDefs";
+
+const logger = LoggerManager.instance.create("OmvPoliticalViewFeatureModifier");
+
+/**
+ * Modifies the MapEnv of the Vector Tiles in Tilezen format with different POV.
+ *
+ * This feature modifier updates feature properties according to different political
+ * point of view.
+ * Political views (or alternate point of views) are supported in Tilezen by adding
+ * additional property with ISO 3166-1 alpha-2 compliant country posix to __default__ property name.
+ * For example country borders (__boundaries__ layer) may have both __kind__ property for
+ * default (major POV) and __kind:xx__ for alternate points of view. This way disputed borders
+ * may be visible or not for certain regions and different users (clients).
+ */
+export class OmvPointOfViewFeatureModifier extends OmvGenericFeatureModifier {
+    private readonly m_countriesPov: string[];
+
+    /**
+     * C-tor.
+     *
+     * @param description
+     * @param respectedPov The codes (in ISO 3166-1 alpha-2 format) of countries which
+     * point of view should be taken firstly into account. The position on the lists
+     * relates to priority, so the first entries have precedence before the later.
+     */
+    constructor(description: OmvFeatureFilterDescription, respectedPov: string[]) {
```

Comment:
Why is respectedPov an array?

Reply:
Hm... firstly I was thinking about support for multiple POVs at once, but probably it is not needed... I will finally remove it... We just need to decide which implementation variant we will choose. I am preparing second PR which is based on MapEnv (feature environment) which will support localized (or political) views.